sql/2019/10_SEO/10_11.sql

```diff
@@ -0,0 +1,26 @@
+#standardSQL
+
+# Linking - fragment URLs (together with SPAs to navigate content)
+# todo: include if is SPA (react/vue) or not (see uncommented lines)
```

Comment:
Suggestion:

```sql
#standardSQL
SELECT
  app,
  APPROX_QUANTILES(
    CAST(JSON_EXTRACT_SCALAR(JSON_EXTRACT_SCALAR(payload, '$._almanac'), "$['seo-anchor-elements'].navigateHash") AS INT64),
    10) AS hash_links
FROM
  `httparchive.pages.2019_07_01_*`
JOIN
  `httparchive.technologies.2019_07_01_*`
USING (_TABLE_SUFFIX, url)
WHERE
  app IN ('React', 'Angular', 'Vue.js')
GROUP BY
  app
```

Needs to be tweaked to extract the particular metric you're after. FWIW ~90% of sites have a `navigateHash` value of 0.